Rakefile

```diff
@@ -4,6 +4,7 @@ require 'rake/testtask'
 require 'yaml'
 require 'yajl'
 require 'open-uri'
+require 'json'
```

Comment:
I can't see any reason for this addition.  Is there a reason for this?  If not, can you please remove it?